benchmarks/lasso/solvers/gapsafe.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+
+from benchopt.base import BaseSolver
+from benchopt.util import safe_import
+
+
+with safe_import():
+    from gsroptim.lasso import lasso_path  # TODO make it available in init?
+
+
+class Solver(BaseSolver):
+    name = 'Gap Safe'
```

Comment:
For now, when using `-s gapsafe`, this is checking that `name.lower == 'gapsafe'` so this fails because of the 'space'.

The error you see is simply that the `DataFrame` is empty. I will try to put a better error report and raise an error when no solver is selected.

Reply:
`-s safe` should work.

Reply:
Ah, I see. I changed the name to `gapsafe` 
I wasn't able to determine the cause of the error by myself so showing an error when an unknown solver is passed is probably a good idea!

Reply:
Also having two separate fields : `id` that is matchable with `-s` and `name` or `display_name`? WDYT?